asmcomp/cmmgen.ml

```diff
@@ -601,7 +601,7 @@ let get_field env ptr n dbg =
         else Mutable
       | _ -> Mutable
   in
-  Cop(Cload (Word_val, mut), [field_address ptr n dbg], dbg)
+  Cop(Cload {memory_chunk=Word_val; mutability=mut; is_atomic=false}, [field_address ptr n dbg], dbg)
```

Comment:
Use punning here as well. Rename the `mut` variable as `mutability` and you can then write the record as `Cload {memory_chunk=Word_val; mutability; is_atomic=false}`.